packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -134,55 +137,76 @@ export const formatPriceEvents = async (
         ancillaryData: event.args.ancillaryData,
         proposedPrice: ethers.utils.formatEther(event.args.proposedPrice),
         eventIndex: event.logIndex,
+        oracleAddress: event.address,
       };
     })
   );
 };
 
 export const getPolymarketMarkets = async (params: MonitoringParams): Promise<PolymarketMarket[]> => {
+  const markets = [];
+  const pagination = 100;
+  let offset = 0;
+
   const whereClause =
     "uma_resolution_status!='settled'" +
     " AND uma_resolution_status!='resolved'" +
     " AND question_ID IS NOT NULL" +
     " AND clob_Token_Ids IS NOT NULL";
 
-  const query = `
-    {
-      markets(where: "${whereClause}") {
-        resolvedBy
-        questionID
-        createdAt
-        question
-        outcomes
-        outcomePrices
-        liquidityNum
-        volumeNum
-        clobTokenIds
-        endDate
-        umaResolutionStatus
+  let moreMarketsAvailable = true;
+  while (moreMarketsAvailable) {
+    const query = `
+      {
+        markets(where: "${whereClause}", limit: ${pagination}, offset: ${offset}) {
+          resolvedBy
+          questionID
+          createdAt
+          question
+          outcomes
+          outcomePrices
+          liquidityNum
+          volumeNum
+          clobTokenIds
+          endDate
+          umaResolutionStatus
+        }
       }
+    `;
+
+    const { markets: polymarketContracts } = (await request(params.graphqlEndpoint, query)) as any;
```

Comment:
nit: Could we use a specific type here that mirrors the types that were requested in the graphql query above? This would get us a bit more type validation below.